frontend/src/modules/LocalizeRedux.js

```diff
@@ -1,12 +1,36 @@
+import LOCALIZE from "../text_resources";
+
+export const LANGUAGES = {
+  english: "en",
+  french: "fr"
+};
+
 // Action Types
 export const SET_LANGUAGE = "localize/SET_LANGUAGE";
 
 // Action Creators
-const setLanguage = language => ({ type: SET_LANGUAGE, language });
+const setLanguage = language => {
+  // Save the language preference in the browser so we don't have
+  // to ask again.
+  localStorage.setItem("catLanguage", language);
```

Comment:
Here, we're storing the selected language in local storage, that's perfect. However, we never delete it. 

First, I think that we should delete it on logout action, so it "resets" the home page when somebody logout. But it may also be deleted on the last action after the eMIB sample test completion. 

Secondly. imagine using a test computer or something like that, so the first user would come and select "English" for example. Complete the test... and leave. Then a new candidate uses our application the next day, but the computer still running or did not have its cookies cleaned up. The selected language will still be "English" and the second user will not have the language page as the home page.

Reply:
Good idea. I deleted it on test de-activation because I think that makes the most sense for right now, but we can always add it elsewhere.